Doppler.ContactPolicies.Business.Logic/Extensions/MapperExtension.cs

```diff
@@ -27,5 +22,21 @@ public static class MapperExtension
                     : new List<ExcludedSubscribersLists>(contactPoliciesSettings.ExcludedSubscribersLists)
             };
         }
+
+        public static ContactPoliciesSettings ToDao(
+            this ContactPoliciesSettingsDto contactPoliciesSettings)
+        {
+            if (contactPoliciesSettings is null)
+                return null;
+            return new ContactPoliciesSettings
+            {
+                Active = contactPoliciesSettings.Active,
+                EmailsAmountByInterval = contactPoliciesSettings.EmailsAmountByInterval,
+                IntervalInDays = contactPoliciesSettings.IntervalInDays,
+                ExcludedSubscribersLists = contactPoliciesSettings.ExcludedSubscribersLists is null
+                    ? null
+                    : new List<ExcludedSubscribersLists>(contactPoliciesSettings.ExcludedSubscribersLists)
```

Comment:
Would it be the same?

```suggestion
                ExcludedSubscribersLists = contactPoliciesSettings.ExcludedSubscribersLists?.ToList()
```

Reply:
actually are the same, but when looking at that I realized that it was not doing what I wanted, since the front can send me a null and in the repository query it cannot enter as null, it has to be like an empty list. I will push the change for that one, **thanks** @andresmoschini